packages/form-layout/src/vaadin-form-item.d.ts

```diff
@@ -89,7 +89,8 @@ import { FormItemMixin } from './vaadin-form-item-mixin.js';
  * ---|---|---
  * `--vaadin-form-item-label-width` | Width of the label column when the labels are aside | `8em`
  * `--vaadin-form-item-label-spacing` | Spacing between the label column and the input column when the labels are aside | `1em`
- * `--vaadin-form-item-row-spacing` | Height of the spacing between the form item elements | `1em`
+ * `--vaadin-form-item-row-spacing` | (DEPRECATED: Use `--vaadin-form-layout-row-spacing` on `<vaadin-form-layout>`
+ * instead) Height of the spacing between the form item elements | `1em`
```

Comment:
Let's keep the entire description text on the same line, as otherwise it disrupts the table formattting in VS Code:

![image](https://github.com/user-attachments/assets/a906afa0-bd4a-4783-853f-c8902c44e6e6)

Reply:
Done.